DXRFixes/DeusEx/Classes/ScriptedPawn.uc

```diff
@@ -663,6 +692,33 @@ state Sitting
 
 }
 
+// returns true if the damage type can knock the pawn out
+function bool CanKnockUnconscious(coerce string damageType)
+{
+    if (Robot(self) != None) {
+        return false;
+    }
+
+    // small creatures can't survive being hit by JC
+    if (mass <= 10.0) {
```

Comment:
we're not changing the behavior of other mods, only DXRando vanilla

```suggestion
    if (mass <= 10.0 && #defined(injections)) {
```

Reply:
could cats survive these things but not birds or fish? we could always do something like

```suggestion
    if (mass <= 10.0 && mass > 5 && #defined(injections)) {
```